### ISSUE TITLE
Model Training with gradient exploding

### ISSUE THREAD

**Issue by dddraxxx** [AI-Application-and-Integration-Lab/SAM4MLLM#3]
Hi, thanks for you great work!

When I was trying to reproduce the results using your training code, I found that the loss is exploding after a while. I wonder if you know what is the problem? Did you use the exact same code to train your model in paper?

Thanks a lot!

<img width="1794" alt="Image" src="https://github.com/user-attachments/assets/93edbc41-658b-4a99-9a73-65b9e2980db7" />

**Comment by dddraxxx** [comment]
I am using 8 card GPU with exactly your learning rate and batch size setting in the script
